alf/environments/suite_socialbot.py

```diff
@@ -74,6 +74,10 @@ def load(environment_name,
         A PyEnvironmentBase instance.
     """
     _unwrapped_env_checker_.check_and_update(wrap_with_process)
+    if gym_env_wrappers is None:
+        gym_env_wrappers = ()
+    if env_wrappers is None:
+        env_wrappers = ()
```

Comment:
Why do you need this? The default is already `()`

Reply:
This is for command line argument override for gin configs.
--gin_param="xxx=()"
Parens can be mis-interpreted by the linux shell, passing in None is easier.